modules/internal/String.chpl

```diff
@@ -936,12 +944,12 @@ module String {
                   done = true;
                 // no limit
                 } else {
-                  chunk = localThis[chunkStart..i];
+                  chunk = localThis[chunkStart..i-1];
```

Comment:
Why i - 1 here? Wouldn't it need to be subtracting a codepoint size?

Reply:
`i` was zero-based and is now one-based, so now we need `i-1`.

Reply:
To clarify, `i` indexes the *last* byte of the *previous* character, which is why we only need to subtract 1 and not the codepoint size.